src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
Creating a new instance of Random every time the method is called is inefficient. Consider using a single instance as a class-level field instead.